project2/index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Playfair+Display&display=swap" rel="stylesheet">
+    <title>Document</title>
```

Comment:
You should always update the `title` here to be relevant to your page/site. This element controls how your site/page is identified in browser tabs, history and bookmarks:
<img width="489" alt="Screen Shot 2019-10-28 at 10 01 11 AM" src="https://user-images.githubusercontent.com/2653980/67684650-f3486100-f969-11e9-9b13-29839265f244.png">